src/gui/C4LoaderScreen.h

```diff
@@ -13,29 +13,45 @@
  * To redistribute this file separately, substitute the full license texts
  * for the above references.
  */
-// startup screen
+ // startup screen
 
 #ifndef INC_C4LoaderScreen
 #define INC_C4LoaderScreen
 
 #include "graphics/C4FacetEx.h"
 
+
 class C4LoaderScreen
 {
 public:
+	enum Flag {
```

Comment:
Should be `enum class` as you're always using `Flag::FOO` anyways.

Reply:
This would reqire C4LoaderScreen Flags to be strongly typed - requiring all the bitwise-operators that one has learned to love needed to be reimplemented. I guess it's less effort this way.

Reply:
Ah. It looked like you did that anyways.

Reply:
Yup, I did. Then I noticed in my latest changes that it isn't needed this way, so I removed the superfluous methods.